### PR TITLE
Updates to get the repo up to spec, fix up docs, small bug fix

### PR DESCRIPTION
**Pull request by bmtcril**
A few things left over from reviews on #1  and small cleanup. There's a lot more to do, but this should suffice for a version 0.1 tag.

### CODE REVIEW

**Comment by bmtcril** [comment]
@feanil I poked you on this one just to review the last commit which has all of the cookiecutter stuff to get the repo up to current state so I can then add in things like the requirements checks.

**Comment by pomegranited** [comment]
@bmtcril 
> but this should suffice for a version 0.1 tag.

Oh bugger, I already pushed a 0.1 tag, sorry! Feel free to overwrite it after this merges though, since we're the only ones using this repo at the moment.

**Comment by pomegranited** [comment]
@bmtcril Oh and oops.. I just realised that I neglected to pin `clickhouse-connect>0.5,<0.6` here [like you asked me to do](https://github.com/openedx/xapi-db-load/pull/1#discussion_r1094599921) :( Could you do that here?

File: catalog-info.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  annotations:
+    # (Optional) Annotation keys and values can be whatever you want.
+    # We use it in Open edX repos to have a comma-separated list of GitHub user
+    # names that might be interested in changes to the architecture of this
+    # component.
+    openedx.org/arch-interest-groups: ""
+spec:
+
+  # (Required) This can be a group(`group:<group_name>` or a user(`user:<github_username>`)
+  owner: ""
```
Comment (feanil) [inline catalog-info.yaml:18]: Do you need a github team created for the people that will own this repo?

File: catalog-info.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+  # (Required) Acceptable Type Values: service, website, library
+  type: ''
```
Comment (feanil) [inline catalog-info.yaml:21]: I think probably `library` for this until we add more granular types.

File: catalog-info.yaml [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+  # (Required) Acceptable Lifecycle Values: experimental, production, deprecated
+  lifecycle: 'experimental'
+
+  # (Optional) The value can be the name of any known component.
+  subcomponentOf: '<name_of_a_component>'
+
+  # (Optional) An array of different components or resources.
+  dependsOn:
+  - '<component_or_resource>'
+  - '<another_component_or_resource>'
```
Comment (feanil) [inline catalog-info.yaml:32]: I think you can just drop all of this.

File: docs/conf.py [hunk 4]
```diff
@@ -0,0 +1,515 @@
...
+
+All configuration values have a default; values that are commented out
+serve to show the default.
+"""
+import os
+import re
+import sys
+from subprocess import check_call
+
+import edx_theme
```
Comment (feanil) [inline docs/conf.py:19]: The cookiecutter is still not updated unfortunately, but you should drop usage of the edx-sphinx-theme using the steps described here: https://docs.openedx.org/en/latest/developers/how-tos/switch-to-the-sphinx-book-theme.html

You can do that as a follow-up if you want.

**Review by feanil — COMMENTED** [review]
A few comments on the cookiecutter part.

File: catalog-info.yaml (hunk 1 above)
Comment (bmtcril) [reply catalog-info.yaml]: There was already one created. I've touched so many versions of this file I just forgot to update the one in this repo, but I think it's good now.

**Comment by bmtcril** [comment]
@feanil I think this is ready for re-review

**Review by feanil — APPROVED** [review]
Metadata and maintenance stuff looks good.

**Review by pomegranited — APPROVED** [review]
:+1: Works for me, thanks @bmtcril !

That's some crazy quantity of boilerplate though, hey..

- [x] I tested this using this branch with my https://github.com/openedx/tutor-contrib-clickhouse/pull/1
- [x] I read through the code
- [x] <strike>I checked for accessibility issues</strike> N/A
- [x] Includes documentation
- [x] Commit structure follows [OEP-0051](https://github.com/edx/open-edx-proposals/pull/189)

**Pull request merged**